### PR TITLE
Add active admin with basic auth

### PR DESCRIPTION
**Pull request by dmytsuu**

### CODE REVIEW

File: app/controllers/concerns/accessible.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Accessible
+  extend ActiveSupport::Concern
+
+  protected
+
+  def authenticate_admin
+    authenticate_or_request_with_http_basic do |username, password|
+      username == Rails.application.credentials[:admin_username] && \
```
Comment (viktor-shmigol) [inline app/controllers/concerns/accessible.rb]: Move to ENV instead of credentials

File: app/inputs/inet_input.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```
Comment (viktor-shmigol) [inline app/inputs/inet_input.rb]: Do we need this?

Comment (dmytsuu) [reply app/inputs/inet_input.rb]: raises exception without input files in active admin forms

File: app/models/user.rb [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
-  attr_accessor :registration_token
   extend Devise::Models
   devise :database_authenticatable,
-         :registerable,
          :validatable,
          :omniauthable
   include DeviseTokenAuth::Concerns::User
 
-  validates :email, uniqueness: true, allow_nil: true
+  validates :email, uniqueness: true, allow_nil: false
```
Comment (viktor-shmigol) [inline app/models/user.rb]: Remove option allow_nil: false

**Pull request merged**